Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Deleting-and-Restoring-Pages/index.md

```diff
@@ -1,38 +1,47 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # Deleting and Restoring Pages
+In this guide you will get a quick overview of how to delete pages from the section tree.
+Further more will also be shown how you can restore them if you end up changing your mind.
+
+
 
 ## Deleting a Page
 
-If you have pages that you no longer want on the website you are able to delete them. Upon deletion, the page is moved to the Recycle Bin as a safety net just in case you change your mind and wish to restore the page.
+If you have pages that you no longer want on the website you are able to delete them. Upon deletion, the page is moved to the ***Recycle Bin*** as a safety net in case you wish to restore the page.
 
-1. Find the page you wish to delete in the website tree view.
-2. Right click on the page name in the tree view or click the actions link in the top right when on the selected page.
+1. Find the page you wish to delete in the section tree view.
+2. Right click on the page name in the tree view or click the actions button in the top right when on the selected page.
 3. Click ***Delete.***
 4. Click ***OK*** to confirm you wish to delete the page or click ***Cancel*** if you do not.
 
 ## Restoring a Deleted Page from the Recycle Bin
 
-The Recycle Bin is a separate tree list which can be found at the bottom of the content tree view. Clicking the arrow to the left of the Recycle Bin icon will reveal any pages that have been deleted from the website. Restoring these pages follows the same process as moving a page between sections.
+The ***Recycle Bin*** is a separate tree list which can be found at the bottom of the section tree view. Clicking the arrow to the left of the Recycle Bin icon will reveal any pages that have been deleted from the website. Restoring these pages follows the same process as moving a page between sections.
 
 1. Click on the ***(•••)*** icon for the page you wish to restore from the Recycle Bin to reveal the Actions menu.
 2. Click on ***Move*** and select the page you wish to nest the page under.
+
+    You can also choose to ***Restore***, this will restore the page to where it was before you deleted it.
 3. Click on the ***Move*** button to confirm the move and then you will see a confirmation message.
+
+    If you chose to ***Restore*** then you click the ***Restore*** button to confirm the restoration of the page.
```

Comment:
Having these lines that not really a part of the list, makes this look a bit strange 🤔 Like, there's too much space between the lines compared to the other lists.

Not a big thing, but if you have some time, it would be great if you could look into it! 😁

Reply:
will see what i can do